.github/workflows/CI.yml

```diff
@@ -157,20 +159,38 @@ jobs:
             -DSUNSHINE_CONFIGURE_ONLY=ON \
             ..
 
+      - name: Debug Manifest
+        working-directory: build
+        run: |
+          cat ${APP_ID}.yml
+
       - name: Build Linux Flatpak
         working-directory: build
         run: |
-          sudo su $(whoami) -c 'flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean \
-            --stop-at=cuda build-sunshine dev.lizardbyte.sunshine.yml'
+          sudo su $(whoami) -c "flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean \
+            --stop-at=cuda build-sunshine ${APP_ID}.yml"
           cp -r .flatpak-builder copy-of-flatpak-builder
-          sudo su $(whoami) -c 'flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean \
-            build-sunshine dev.lizardbyte.sunshine.yml'
+          sudo su $(whoami) -c "flatpak run org.flatpak.Builder --arch=${{ matrix.arch }} --repo=repo --force-clean \
+            build-sunshine ${APP_ID}.yml"
           rm -rf .flatpak-builder
           mv copy-of-flatpak-builder .flatpak-builder
-          sudo su $(whoami) -c 'flatpak build-bundle --arch=${{ matrix.arch }} ./repo \
-            ../artifacts/sunshine_${{ matrix.arch }}.flatpak dev.lizardbyte.sunshine'
-          sudo su $(whoami) -c 'flatpak build-bundle --runtime --arch=${{ matrix.arch }} ./repo \
-            ../artifacts/sunshine_debug_${{ matrix.arch }}.flatpak dev.lizardbyte.sunshine.Debug'
+          sudo su $(whoami) -c "flatpak build-bundle --arch=${{ matrix.arch }} ./repo \
+            ../artifacts/sunshine_${{ matrix.arch }}.flatpak ${APP_ID}"
+          sudo su $(whoami) -c "flatpak build-bundle --runtime --arch=${{ matrix.arch }} ./repo \
+            ../artifacts/sunshine_debug_${{ matrix.arch }}.flatpak ${APP_ID}.Debug"
+
+      - name: Lint Flatpak
+        working-directory: build
+        run: |
+          echo "Linting flatpak manifest"
+          flatpak run --command=flatpak-builder-lint org.flatpak.Builder --exceptions manifest ${APP_ID}.yml
+
+          echo "Linting flatpak repo"
+          mirror_url="https://dl.flathub.org/media/dev/lizardbyte/app.Sunshine/453e7e848f225b26ce4d5f2500a806c1/screenshots/image-1_orig.webp"
```

Comment:
Thank you!

I have one question. Is the "--exceptions" arg only going to allow exceptions that are for this flatpak id? Or it will pass any if the rule says exceptions allowed?

Reply:
I must have missed something.

```txt
usage: flatpak-builder-lint [-h] [--version] [--exceptions] [--appid APPID]
                            [--cwd] [--ref REF]
                            {builddir,repo,manifest,appstream} path
flatpak-builder-lint: error: unrecognized arguments: --mirror-screenshots-url=https://dl.flathub.org/media
```